jenkinsfiles/RHEL_Check.groovy

```diff
@@ -116,18 +119,23 @@ pipeline {
 
     stages {
         stage('Check_User_in_Org') {
-            agent {
-                label "master"
-            }
             steps {
                 script {
                     try {
                         retry(2) {
                             fill_task_name_description()
-                            def check_user_return = sh(script: "python3 /localdisk2/oneDPL_CI/check_user_in_group.py -u  ${env.User}", returnStatus: true, label: "Check User in Group")
+                            def check_user_return = sh(script: "python3 /export/users/oneDPL_CI/check_user_in_group.py -u  ${env.User}", returnStatus: true, label: "Check User in Group")
                             echo "check_user_return value is $check_user_return"
                             if (check_user_return == 0) {
                                 user_in_github_group = true
+                                if (env.OneAPI_Package_Date == "Default") {
+                                    sh(script: "bash /export/users/oneDPL_CI/get_good_compilor.sh ", label: "Get good compiler stamp")
```

Comment:
Typo: `get_good_compilor.sh` -> `get_good_compiler.sh`
Please fix here and everywhere below